src/main/java/blockrenderer6343/ClientProxy.java

```diff
@@ -18,7 +26,9 @@ public void preInit(FMLPreInitializationEvent event) {
         GuiContainerManager.addTooltipHandler(new InputHandler());
         if (BlockRenderer6343.isGTLoaded) {
             GregTechAPI.addDummyWorld(TrackedDummyWorld.class);
+            GregTechAPI.addDummyWorld(ObserverWorld.class);
```

Comment:
The ObserverWorld is used to determine the tier for structures such as nano forge & gorge that have tiered structures but no tiered blocks. It works slightly differently and I didn't feel like shoving all of that code into the same world used for rendering since it'll never be useful there.